components/user/UserTabs.js

```diff
@@ -49,28 +49,28 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           return aVal.toLowerCase() > bVal.toLowerCase()
             ? 1
             : aVal.toLowerCase() < bVal.toLowerCase()
-            ? -1
-            : 0;
+              ? -1
```

Comment:
These are from a formatter(prettier), right?

Reply:
I think these changes should not be in this pull request as it is a separate file with no changes related to this pull request

Reply:
> These are from a formatter(prettier), right?

yes